aospy/test/test_utils_times.py

```diff
@@ -144,57 +144,60 @@ def test_numpy_datetime_range_workaround(self):
         )
 
     def test_numpy_datetime_workaround_encode_cf(self):
+        def create_test_data(
+                days, ref_units, expected_units, expected_date):
```

Comment:
`expected_date` isn't used.

Nit: non-standard to start the func args on the next line.  I would just break after `ref_units` to stay within 80 chars.